### ISSUE TITLE
it is not possible to start new scripts in javascript instance 1

### ISSUE THREAD

**Issue by CKMartens** [ioBroker/ioBroker.javascript#449]
I upgraded to js-controller 2.1.0. I also have two instances of the java script adapter in version 4.3.3. Instance 0 runs on the master ioB01. Instance 1 on the slave ioB02.

When I create a new script on instance 1, the script does not start. No matter if the script is created with blockly or javascript. It also appears in the log as just under the javascript.0 as stops:
`javascript.0 2019-11-18 07: 15: 19.383 info (1606) Stop script script.js.Heating.tadoBoost`

Even if the script is created in instance 0 - where the script works - and then moved to instance 1, the script then stops working.

Scripts created and launched before the js-controller upgrade in instance 1 are not affected.

**Comment by GermanBluefox** [comment]
In deed: https://github.com/ioBroker/adapter-react/blob/master/src/Connection.js#L392 that could be a Problem

**Comment by CKMartens** [comment]
I tried to move the script to punch 1 after doing the hotfix suggested in the forum.

The script starts, ioBroker shows no output in the log. Per console on the slave is the script is started. The data points that the script should create with createState are created but can not be edited. There is also no output via console.log (). The subscription with on ({id: xxx}) is created according to log

```
2019-11-18 14: 35: 23.862 - info: javascript.1 (1432) script.js.Heat.tadoBoost: tadoBoost: Script started
2019-11-18 14: 35: 23.863 - info: javascript.1 (1432) script.js.Heating.tadoBoost: registered 5 subscriptions and 0 schedules

```

But if the data point to which the subscription is being accessed, the script will not respond.

If I move the script back to instance 0, it works fine.

**Comment by Apollon77** [comment]
which hotfix you did? maybe try to patch the one line in javascript sandbox.js

**Comment by CKMartens** [comment]
I added the solution you gave in ioBroker forum. And i added javascript from github with the changes in sandbox.js.
But th problem still exist.

**Comment by CKMartens** [comment]
I documatet a short test:
I write a simple script in instance 1:
![grafik](https://user-images.githubusercontent.com/19702758/69125850-094bcc00-0aa7-11ea-8e9a-85276a05c631.png)
![grafik](https://user-images.githubusercontent.com/19702758/69125883-1f598c80-0aa7-11ea-887f-845e0c6ec62d.png)

After start in ioBroker-Admins log section is no output from javascript.1. Only in the log-file on the slave a saw:
![grafik](https://user-images.githubusercontent.com/19702758/69125956-562fa280-0aa7-11ea-9b0f-692d3a112a0c.png)
The datapoint was written:
![grafik](https://user-images.githubusercontent.com/19702758/69125987-69427280-0aa7-11ea-9eba-d0d32bf267bb.png)
But when i triggert the test-state nothing happend. No 'hello World' in log, no set of the state to false.

After i change to instance 0 i have output in the log section:
![grafik](https://user-images.githubusercontent.com/19702758/69126139-bd4d5700-0aa7-11ea-9a70-b217cbcdd7fe.png)

an the simple script works well.

**Comment by Apollon77** [comment]
shpuld be fixed by js-controller 2.1.1 too

**Comment by CKMartens** [comment]
Hallo @Apollon77 and @GermanBluefox very good job!

i installed js-controller 2.1.1and javascript 4.3.4. And first test with the short script above looks fine. Thank you.
